posthog/test/test_migration_0893.py

```diff
@@ -0,0 +1,139 @@
+        DataWarehouseTable: DataWarehouseTableModel = apps.get_model("posthog", "DataWarehouseTable")
+
+        self.organization = Organization.objects.create(name="o1")
+        self.project = Project.objects.create(organization=self.organization, name="p1", id=1000001)
+        self.team = Team.objects.create(organization=self.organization, name="t1", project=self.project)
+
+        with freeze_time("2025-01-01T12:00:00.000Z"):
+            source_1 = ExternalDataSource.objects.create(team=self.team, source_type="Stripe")
+
+            self.table_1 = DataWarehouseTable.objects.create(
+                team=self.team, name="table_a", external_data_source_id=source_1.id
+            )
+
+            # tests that deleted tables are ignored
+            self.schema_1 = ExternalDataSchema.objects.create(
+                team=self.team,
+                table=self.table_1,
+                source_id=source_1.id,
+                deleted=True,
+            )
+            # tests that sync configs with partitioning disabled are ignored
+            self.schema_2 = ExternalDataSchema.objects.create(
+                team=self.team,
+                table=self.table_1,
+                source_id=source_1.id,
+                sync_type_config=SYNC_TYPE_PARTITIONING_DISABLED,
+            )
+            # tests that sync configs with md5 partitions are ignored
+            self.schema_3 = ExternalDataSchema.objects.create(
+                team=self.team,
+                table=self.table_1,
+                source_id=source_1.id,
+                sync_type_config=SYNC_TYPE_PARTITIONING_MD5,
+            )
+            # tests that sync configs with numerical partitions are ignored
+            self.schema_4 = ExternalDataSchema.objects.create(
+                team=self.team,
+                table=self.table_1,
+                source_id=source_1.id,
+                sync_type_config=SYNC_TYPE_PARTITIONING_NUMERICAL,
+            )
+            # tests that sync configs with partition format = "day" are ignored
+            self.schema_5 = ExternalDataSchema.objects.create(
+                team=self.team,
+                table=self.table_1,
+                source_id=source_1.id,
+                sync_type_config=SYNC_TYPE_PARTITIONING_DAY,
+            )
+            # tests that sync configs with partition format = "month" are ignored
+            self.schema_6 = ExternalDataSchema.objects.create(
+                team=self.team,
+                table=self.table_1,
+                source_id=source_1.id,
+                sync_type_config=SYNC_TYPE_PARTITIONING_MONTH,
+            )
+            # tests that sync configs which match our target set have partition format set to "week"
+            self.schema_7 = ExternalDataSchema.objects.create(
+                team=self.team,
+                table=self.table_1,
+                source_id=source_1.id,
+                sync_type_config=SYNC_TYPE_PARTITIONING_MONTH,
```

Comment:
This should be using `SYNC_TYPE_AFFECTED_SET` instead of `SYNC_TYPE_PARTITIONING_MONTH`